scrapers/csc_department_scraper.py

```diff
@@ -0,0 +1,88 @@
+from requests.models import codes
+from models import Professor, Room
+from autoscraper import AutoScraper
+import os
+from scraper import registry, Scraper
+from scrapers.departmentscraper import DepartmentScraper
+from bs4 import BeautifulSoup as bsp
+import requests
+from urllib.request import Request, urlopen  # TODO: temp
+import re
+
+# 1. Register your scraper so that it gets run by the scrape_data file. Make sure your class is a subclass of `Scraper`
+@registry.register
+class CSCDepartmentScraper(DepartmentScraper):
+    """
+    Fetches information about Cal Poly Music Department professors
+    """
+
+    links = {
+        "Computer Science and Software Engineering": "https://csc.calpoly.edu/faculty/"
+    }
+
+    # 2. The only requirement of the class is the `scrape()` function. This will get data from the internet and upload it to the server.
+    def scrape(self):
+        site = "https://csc.calpoly.edu/faculty/"
+        hdr = {"User-Agent": "Mozilla/5.0"}
+        req = Request(site, headers=hdr)
+        page = urlopen(req)
```

Comment:
Using the `requests` package might be easier. You can call requests.get(url) and it will figure out the headers for you.